### ISSUE TITLE
Mosquitto broker log should show the timestamp in a readable format instead of Unix timestamp in miliseconds

### ISSUE THREAD

**Issue by elbueno222** [home-assistant/addons#1802]
## The problem

Mosquitto broker logs shows the time in miliseconds instead of DD:MM:YYYY HH:MM:SS

## Environment

- Add-on with the issue: Mosquitto broker
- Add-on release with the issue: 
- Last working add-on release (if known): 
- Operating environment (OS/Supervised): 

## Problem-relevant configuration

```yaml

```

## Traceback/Error logs

```txt

```

**Comment by elbueno222** [comment]
Can someone be assigned to this issue?

**Comment by cabletie** [comment]
+1 on that ...

**Comment by mpasteur** [comment]
Hi,
This is on mosquitto's side not HA but you can add these lines to your customization config:
```
log_timestamp true
log_timestamp_format %Y-%m-%dT%H:%M:%S

```
then after a restart you'll see:
```
2021-04-03T19:53:11: Opening ipv4 listen socket on port 1883.
2021-04-03T19:53:11: Opening ipv6 listen socket on port 1883.
2021-04-03T19:53:11: Opening websockets listen socket on port 1884.

```
regards,
maybe a PR to add this to the addon's default conf  will be accepted ...

**Comment by elbueno222** [comment]
@mpasteur Where is that customization config? I tried to add this to the Mosquito broker configuration with no luck:

´´´
log_timestamp: true
log_timestamp_format: "%Y-%m-%dT%H:%M:%S"
´´´

**Comment by mpasteur** [comment]
You've to enable the flag "customize" in the configuration as explained in the addon's documentation:

customize:
  active: true
  folder: mosquitto

then you create a file 'logging.conf' (can be another name but .conf is mandatory i think) on the /share/mosquitto folder.
Put the two lines on this file.

**Comment by elbueno222** [comment]
I tried to do that but unfortunately the folder /share/mosquitto doesn´t exist, and I tried to find it with the command ```find / -type d -name "mosquitto"``` but it doesn´t find the mosquitto folder anywhere

**Comment by elbueno222** [comment]
@mpasteur IT WORKS!! I just had to create the mosquitto folder within /share and then add the mosquitto.conf there and it works. Thank you very much

**Comment by Heisenberg2980** [comment]
> Hi, This is on mosquitto's side not HA but you can add these lines to your customization config:
> 
> ```
> log_timestamp true
> log_timestamp_format %Y-%m-%dT%H:%M:%S
> ```
> 
> then after a restart you'll see:
> 
> ```
> 2021-04-03T19:53:11: Opening ipv4 listen socket on port 1883.
> 2021-04-03T19:53:11: Opening ipv6 listen socket on port 1883.
> 2021-04-03T19:53:11: Opening websockets listen socket on port 1884.
> ```
> 
> regards, maybe a PR to add this to the addon's default conf will be accepted ...

For everyone who applied this workaround, issue has been properly fixed now in latest version of the add-on 6.1.3 (the issue raised was https://github.com/home-assistant/addons/issues/2630) so the two lines mentioned before must be removed from the file in the mosquitto folder, otherwise the add-on won´t start after the update
